airgun/views/computeresource.py

```diff
@@ -440,6 +443,7 @@ class storage(RemovableWidgetsItemsListView):
             add_item_button = Text(
                 "//button[contains(concat(' ', @class, ' '), ' btn-add-controller ')]"
             )
+            data_store = FilteredDropdown(id="s2id_autogen41")
```

Comment:
Does this have any other locator? This looks like a temporary one.

Reply:
change the temporary locator.